appender-log4j/src/main/java/com/van/logging/log4j/Log4jAppender.java

```diff
@@ -179,6 +181,30 @@ public void setS3SseKeyType(String s3SseKeyType) {
         getS3Configuration().setSseConfiguration(sseConfig);
     }
 
+    /**
+     * private | public-read | public-read-write | authenticated-read | bucket-owner-read | bucket-owner-full-control | log-delivery-write
+     *
+     * @param acl
+     */
+    public void setS3CannedAcl(String acl) {
```

Comment:
I don't think we need to support these variants. The S3 SDK's `CannedAccessControlList ` enum is the authoritative source of the value space.

```
try {
    getS3Configuration().setCannedAclFromValue(acl);
} catch (IllegalArgumentException ex) {
    System.err.println(String.format("Ignoring unrecognized canned ACL value %s", acl));
}
```